Week-1/Exercise-1.1/Readme.md

```diff
@@ -0,0 +1,95 @@
+# How does a Browser work
```

Comment:
Good research and analysis. Good job. 

Can we talk a bit more on how the DOM is created? (HTML + CSS DOM) ?